google/codelab/provision_spinnaker_codelab_image.sh

```diff
@@ -58,6 +58,7 @@ sudo -u jenkins -H sh -c '/home/jenkins/aptly publish repo -architectures="amd64
 
 # as jenkins, configure jenkins config directory
 # this storage bucket is public so we can pull the jenkins config from anywhere
+cd /var/lib/jenkins
```

Comment:
In the other PR you changed /var/lib/jenkins to /home/jenkins.
Is this /var/lib/jenkins intentional?